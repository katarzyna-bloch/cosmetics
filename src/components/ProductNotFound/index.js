import React from 'react'
import styled from 'styled-components'
import PageWrapper from '../PageWrapper'

const Content = styled(PageWrapper)`
  text-align: center;
`

const ProductNotFound = () => (
  <Content>
    <h3>Nie znaleziono produktu :(</h3>
  </Content>
)

export default ProductNotFound
