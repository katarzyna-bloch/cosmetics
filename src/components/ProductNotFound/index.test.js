import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import ProductNotFound from '.'

it('should render correctly', () => {
  const wrapped = shallow(<ProductNotFound />)
  expect(toJson(wrapped)).toMatchSnapshot()
})
