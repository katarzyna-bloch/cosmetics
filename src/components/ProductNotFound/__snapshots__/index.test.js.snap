// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<Styled(styled.div)>
  <h3>
    Nie znaleziono produktu :(
  </h3>
</Styled(styled.div)>
`;
